### ISSUE TITLE
The input string was not in a correct format

### ISSUE THREAD

**Issue by DebugDax** [amakvana/EzYuzu#33]
**Describe the bug**
Opening the program the first time did not throw any error, once I chose my Yuzu directory, this error popped up immediately and now also appears every time I open EzYuzu. I currently have Yuzu 3599 from PineappleEA.

**To Reproduce**
1. Select Directory with Yuzu.exe
2. Clicking Continue just makes EzYuzu expect you to still pick a Yuzu directory

**Screenshots**
![image](https://github.com/amakvana/EzYuzu/assets/6743505/77736689-4ed4-4ff9-b6b1-c4a5f6868288)

```

************** Exception Text **************
System.FormatException: The input string '18.3.7' was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, ReadOnlySpan`1 value, TypeCode type)
   at System.Int32.Parse(String s)
   at EzYuzu.Classes.Yuzu.Detectors.YuzuInstallationStateDetector.GetYuzuInstallationStateAsync(Int32 latestVersionAvailable)
   at EzYuzu.FrmMain.RefreshDetectedYuzuInstallationStateAsync(YuzuBranch detectedYuzuBranch)
   at EzYuzu.FrmMain.FrmMain_LoadAsync(Object sender, EventArgs e)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at InvokeStub_SendOrPostCallback.Invoke(Object, Object, IntPtr*)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

**Desktop (please complete the following information):**
 - OS: Windows 11
  - Version 22H2

**Comment by amakvana** [comment]
Hello,

Thanks for opening the issue. It looks like the `version` file is malformed. Try this:

1. Close EzYuzu
2. Browse to `C:\User\Micro\Desktop\yuzuea`
3. Locate the file called `version` & delete it 
4. Relaunch EzYuzu & update

Let me know how this goes

**Comment by amakvana** [comment]
Also in addition to this, try the build below and see if the issue is rectified. I've added some error handling if the `version` file is malformed

[EzYuzu_Issue33_TestBuild.zip](https://github.com/amakvana/EzYuzu/files/11584357/EzYuzu_Issue33_TestBuild.zip)

**Comment by amakvana** [comment]
Thanks for letting me know! I shall push the changes soon

I will keep this open until the changes have been pushed

**Comment by amakvana** [comment]
Closed - Fixed in v1.6.2.0
https://github.com/amakvana/EzYuzu/releases/latest
